test/functional.js

```diff
@@ -563,6 +563,23 @@ module.exports = {
             });
         });
 
+        it('When enabled, TypeScript is compiled!', (done) => {
+            const config = createWebpackConfig('www/build', 'dev');
+            config.setPublicPath('/build');
+            config.addEntry('main', ['./js/render.ts', './js/index.ts']);
+            config.enableTypeScriptLoader();
+
+            testSetup.runWebpack(config, (webpackAssert) => {
+                // check that ts-loader transformed the ts file
+                webpackAssert.assertOutputFileContains(
+                    'main.js',
+                    'document.getElementById(\'wrapper\').innerHTML = "<h1> Hello World!</h1>";'
+                );
```

Comment:
The original intend was to check that we got something expected in the compiled module. It was just about compilation going through and check that the file was imported.

Did you mean that inside that functional test one can make reference to `document.getElementById('app')` ? It can be done of course. But how is this checked later on? and how will this be usefull for the tests ?